LoopKitUI/CGMManagerSettingsNavigationController.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  CGMManagerSettingsNavigationController.swift
+//  MockKitUI
+//
+//  Created by Nathaniel Hamming on 2021-01-13.
+//  Copyright © 2021 LoopKit Authors. All rights reserved.
+//
+
+import UIKit
+import HealthKit
+import LoopKit
+
+final public class CGMManagerSettingsNavigationController: SettingsNavigationViewController, PreferredGlucoseUnitObserver {
```

Comment:
nit: this doesn't feel like a pattern worth extracting, but rather just boilerplate that happens to be reused across some of the current cgm managers, and could very well be implemented differently in a different cgm manager.

Reply:
I'd recommend keeping this in MockKitUI

Reply:
I asked Nate to do this because for my work needs a specific `SettingsNavigationController` for cgms and one for pumps. We can, of course, implement this plus my work in a `SettingsNavigationController` in each plugin that is specific to that plugin, but if that is the case why do we even have a common `SettingsNavigationController`? Should we do away with the common `SettingsNavigationController` altogether and push into the plugin?